pkg/driver/clickhouse/clickhouse.go

```diff
@@ -96,6 +96,11 @@ func (drv *Driver) databaseName() string {
 	return name
 }
 
+func (drv *Driver) onCluster() bool {
+	onCluster := dbutil.MustParseURL(connectionString(drv.databaseURL)).Query().Get("on_cluster")
```

Comment:
is the `on_cluster=true` URL parameter a standard ClickHouse connection string, or did you invent it?